web/impact/impact/views/algolia_api_key_view.py

```diff
@@ -109,7 +109,7 @@ def _entrepreneur_specific_finalist_filter(roles, request):
 def _entrepreneur_specific_alumni_filter(roles, request):
     if is_entrepreneur(request.user):
         has_current_alum_roles = ProgramRoleGrant.objects.filter(
-            program_role__program__program_status=ENDED_PROGRAM_STATUS,
+            program_role__program__program_status=ACTIVE_PROGRAM_STATUS,
```

Comment:
Not your code (at least, not this time :) ) but in general I would try to make use of the reverse lookup:
```
has_current_alum_roles = request.user.programrolegrant_set.filter(
  program_role__user_role__name=UserRole.ALUM,
  program_role__program__program_status=ACTIVE_PROGRAM_STATUS)
```

Not a blocker.